pocs/camera/fli.py

```diff
@@ -215,7 +216,8 @@ def take_exposure(self,
             seconds = seconds * u.second
 
         if not self._exposure_lock.acquire(blocking=False):
-            message = 'Attempt to start exposure on {} ({}) while exposure in progress! Waiting...'.format(self.name, self.uid)
+            message = 'Attempt to start exposure on {} ({})'.format(self.name, self.uid) + \
```

Comment:
Are the remaining changes in this file just pep8 formatting?

Reply:
Yes.